src/main/java/cloud/eppo/EppoClient.java

```diff
@@ -121,7 +131,7 @@ public void refreshConfiguration() {
   protected EppoValue getTypedAssignment(
       String flagKey,
       String subjectKey,
-      SubjectAttributes subjectAttributes,
+      Attributes attributes,
```

Comment:
Note the `SubjectAttributes` class has been renamed to `Attributes` as they are used with actions too now